pkg/monitor/cluster/apiserverstatuses.go

```diff
@@ -0,0 +1,25 @@
+package cluster
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+	"fmt"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func (mon *Monitor) emitKubeApiServerStatuses(ctx context.Context) error {
+	ds, err := mon.cli.AppsV1().DaemonSets("openshift-apiserver").Get("apiserver", metav1.GetOptions{})
+	if err != nil {
+		return err
+	}
+
+	mon.emitGauge("apiserver.statuses", 1, map[string]string{
+		"available": fmt.Sprintf("%d", ds.Status.NumberAvailable),
+		"desired":   fmt.Sprintf("%d", ds.Status.DesiredNumberScheduled),
```

Comment:
actually, you should probably only emit if available != desired

Reply:
This looks strange. Gauge is "in time counter". There always will be 3 API servers. So `statuses=1, desired=3` looks strange. I know this is following what we did before, but why not:
`apiserver.count={available}` ?